package.json

```diff
@@ -23,7 +23,7 @@
   "author": "GitHub",
   "license": "MIT",
   "dependencies": {
-    "@actions/cache": "^3.1.4",
+    "@actions/cache": "https://github.com/cdce8p/toolkit/raw/cache-dry-run-release/packages/cache/actions-cache-3.1.5.tgz",
```

Comment:
@cdce8p  Now that we've released [cache version 3.2.0](https://www.npmjs.com/package/@actions/cache)
Can you please update this and build and commit again?

Reply:
Done. I did built it again locally but since the `3.2.0` version only included my PR, no changes were necessary.